src/main/java/io/fabric8/elasticsearch/plugin/acl/UserRolesSyncStrategy.java

```diff
@@ -33,40 +33,32 @@ public UserRolesSyncStrategy(SearchGuardRoles roles, String userProfilePrefix, S
         this.kibanaIndexMode = kibanaIndexMode;
         this.expire = String.valueOf(expiresInMillis);
     }
-
+    
     protected void syncFromImpl(OpenshiftRequestContext context, RolesBuilder builder) {
+        String kibIndexName = formatKibanaIndexName(context, kibanaIndexMode);
+        String kibRoleName = formatKibanaRoleName(context);
+        RoleBuilder kibRole = new RoleBuilder(kibRoleName)
+                .setClusterActions(KIBANA_ROLE_CLUSTER_ACTIONS)
+                .setActions(kibIndexName, ALL, KIBANA_ROLE_INDEX_ACTIONS)
+                .setActions(ALL, ALL, KIBANA_ROLE_ALL_INDEX_ACTIONS)
+                .expires(expire);
+        builder.addRole(kibRole.build());
+
+
         //think this can be statically added to roles doc
         if (context.isOperationsUser()) {
             RoleBuilder opsRole = new RoleBuilder(SearchGuardRolesMapping.ADMIN_ROLE)
                     .setClusters(OPERATIONS_ROLE_CLUSTER_ACTIONS)
                     .setActions("?operations?", ALL, OPERATIONS_ROLE_OPERATIONS_ACTIONS)
```

Comment:
is this action still required if we have `ALL`?

Reply:
Agree.  We can remove. I wasn't certain why we ever made this division.  The only possibility is if we desire to distinguish actions from one set of indices vs another set but I presume `ALL` would always be the winner